### DISCUSSION TITLE
Support for .NET6

### DISCUSSION THREAD

**Discussion by tegra** [neon-sunset/Pinecone.NET#47] [General]
Hi,

Due to environmental constraints, we are forced to use .NET6. Is there a plan to support .NET6?

**Comment by neon-sunset** [comment]
Hi @tegra , targeting .NET 6 is definitely an option, quickly checking it seems it will require minimal changes.

**Comment by neon-sunset** [comment]
https://github.com/neon-sunset/Pinecone.NET/releases/tag/1.3.0
